src/Hadrian/Haskell/Cabal/Configured.hs

```diff
@@ -0,0 +1,47 @@
+module Hadrian.Haskell.Cabal.Configured where
+
+import Development.Shake.Classes
+import Hadrian.Package.Type
+import GHC.Generics
+
+data ConfiguredCabal = ConfiguredCabal
+    { dependencies :: [PackageName]
+    , name         :: PackageName
+    , version      :: String
+    -- , packageDesc  :: C.PackageDescription
```

Comment:
Do we need `packageDesc`? If not, let's drop it.

Reply:
Fixed in an upcoming commit.